packages/playground/package.json

```diff
@@ -21,7 +21,10 @@
   "exports": {
     ".": "./dist/src/index.js",
     "./vite": "./dist/src/vite/index.js",
-    "./manifest": "./dist/src/manifest.js",
+    "./manifest": {
+      "default": "./do-not-use-virtual-import",
```

Comment:
This is to have this typed virtual import. Alternative is to add a definition file with a custom module declaration 

like
```ts
declare module "@typespec/playground/manifest" {

   export default ...
}
```

Reply:
but I think with this typescript will not pick it up automatically and it will need to be added to the `types` of tsconfig